src/edu/ucsb/nceas/metacat/healthchecks/StartupRequirementsListener.java

```diff
@@ -228,21 +297,23 @@ protected void abort(String message, Exception e) throws RuntimeException {
 
     /**
      * Check if we can load properties from, and write properties to, 'metacat-site.properties'.
-     * If running in a container (eg in a Kubernetes cluster), then we expect the site properties
-     * file to be read-only, so skip the write check
+     * If running in a container (e.g. in a Kubernetes cluster), then we expect the site properties
+     * file to be read-only, so we skip the write check in that environment.
      *
-     * @param sitePropsFilePath the full path to the 'metacat-site.properties' file
+     * @param sitePropsFilePath the full Path to the 'metacat-site.properties' file
+     * @param defaultProperties a Properties object initialized from the 'metacat.properties' file
      * @throws RuntimeException if any unrecoverable problems are found that should cause startup
      *                          to be aborted
      */
-    private void validateSitePropertiesFileRwAccess(Path sitePropsFilePath)
-        throws RuntimeException {
+    private void validateSitePropertiesFileRwAccess(
+        Path sitePropsFilePath, Properties defaultProperties) throws RuntimeException {
 
-        Properties siteProperties = new Properties();
+        runtimeProperties = new Properties(defaultProperties);
```

Comment:
Is there any reason why we need the `defaultProperties` to initialize the runtimeProperties? If we just use the default constructor to initialize the runtimeProperties object, we can only has one parameter in the method signature.